src/nccl_ofi_net.c

```diff
@@ -1031,10 +1026,17 @@ static ncclResult_t ofi_accept(void *listenComm, void **recvComm)
 	req->lComm = lComm;
 	req->dev = dev;
 
+	remote_ep_name = calloc(1, MAX_EP_ADDR);
+	if (OFI_UNLIKELY(req == NULL)) {
+		NCCL_OFI_WARN("Unable to allocate remote_ep_name");
+		ret = ncclSystemError;
+		goto exit;
+	}
+
 	/* Post a buffer for receiving connection requests */
 	do {
-		rc = fi_trecv(lComm->local_ep, NULL, 0, NULL, FI_ADDR_UNSPEC,
-			      lComm->tag | ~max_tag, 0, &req->ctx);
+		rc = fi_trecv(lComm->local_ep, remote_ep_name, MAX_EP_ADDR, NULL,
```

Comment:
Again, what I read from connect() API, it receives here is not remote_ep_name but self's address.